sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -8,12 +8,26 @@ No dependencies on anything, can build on any architecture and with any compiler
 
 ## Example App
 
+Start Qemu
+
+1. `docker pull keystoneenclaveorg/keystone:dev`
```

Comment:
`keystoneenclaveorg/keystone:master` would be the latest

Reply:
resolved